infra/gcp/lib_gcr.sh

```diff
@@ -149,3 +149,22 @@ function empower_svcacct_to_admin_gcr () {
 
     empower_svcacct_to_admin_gcs_bucket "${group}" "${bucket}"
 }
+
+# Grant the Prow service account access to the same perms as the given GCP
+# service account (for Workload Identity).
+# $1: The Prow service account being empowered
+# $2: The GCP service account to draw powers from
+function empower_prowacct_for_workload_identity() {
+    if [ ! $# -eq 2 -o -z "$1" -o -z "$2" ]; then
+        echo "empower_prowacct_for_workload_identity(prow_acct, gcp_acct) requires 2 arguments" >&2
+        return 1
+    fi
+
+    prow_acct="$1"
```

Comment:
I'd consider separating out these variables into their parts so its more clear, I'd recommend this:

```shell
gke_project=$1
namespace=$2
name=$3
gcp_acct=$4
gcp_project=$5

gcloud iam service-accounts add-iam-policy-binding \
  --role roles/iam.workloadIdentityUser \
  --member "serviceAccount:${gke_project}.svc.id.goog[${namespace}/${name}]" \
  "$gcp_acct" --project "$gcp_project"
```

Reply:
I don;t think we need to go quite this far.

I would position this function as `empower_ksa_to_svcacct(project, k8s_svcacct, gcp_svcacct)`